<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BOM版本对比</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton { 
    color: red ;
    font-size:15px;
}

QLabel { 
    color: blue ;
    font-size:15px;
}
QCheckBox { 
    color: blue ;
    font-size:15px;
}

QTableWidget { 
    color: blue ;
    font-size:15px;
}

QTabWidget { 
    color: blue ;
    font-size:15px;
}

QWidget#tab1 { 
    color: red ;
    font-size:15px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择Excel文件:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="button">
       <property name="text">
        <string>上传</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_2">
       <property name="text">
        <string>下载</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,2,4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>原BOM版本  ： </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,2,4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>目标BOM版本： </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>所有数据</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableWidget" name="table1"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>差异行数据</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTableWidget" name="table2"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>逻辑概述: 1.获取俩个版本的物料清单信息； 2.根据物料号来对比信息是否有差异；</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
